apps/redi-talent-pool/src/components/organisms/JobListingCard.tsx

```diff
@@ -39,46 +77,76 @@ export function JobListingCard({
+              breakpoint="mobile"
             >
-              <Icon
-                icon={isFavorite ? 'heartFilled' : 'heart'}
-                className="job-posting-card__favorite__icon"
-              />
-            </div>
-          )}
-          <Element
-            key="name"
-            renderAs="h3"
-            textWeight="bold"
-            textSize={4}
-            className="job-posting-card__job-title"
-          >
-            {jobTitle}
-          </Element>
-          <Element
-            className="content job-posting-card__company-name"
-            key="location"
-            renderAs="div"
-          >
-            {companyName}
-          </Element>
-          {idealTechnicalSkills?.length > 0 ? (
-            <CardTags
-              items={idealTechnicalSkills}
-              shortList
-              formatter={(skill: string) => topSkillsIdToLabelMap[skill]}
-            />
-          ) : null}
+              <Element
+                key="name"
+                renderAs="h4"
+                textWeight="bold"
+                textSize={4}
+                className="job-posting-card__job-title"
+              >
+                {jobTitle}
+              </Element>
+              <Element
+                className="content job-posting-card__company-name"
+                key="company"
+                renderAs="p"
+              >
+                {companyName}
+              </Element>
+              <CardLocation location={location} remote={remote} />
+              {idealTechnicalSkills?.length > 0 ? (
+                <Element renderAs="div">
+                  <CardTags
+                    items={idealTechnicalSkills}
+                    shortList
+                    formatter={(skill: string) => topSkillsIdToLabelMap[skill]}
+                  />
+                </Element>
+              ) : null}
+            </Columns.Column>
+            <Columns.Column className="job-posting-card__lastColumn">
+              <Element
+                className="content job-posting-card__timeFooterBox"
+                key="company"
+                renderAs="div"
+              >
+                {toggleFavorite && (
+                  <div
+                    className="job-posting-card__favorite"
+                    onClick={handleFavoriteClick}
+                  >
+                    <Icon
+                      icon={isFavorite ? 'heartFilled' : 'heart'}
+                      className="job-posting-card__favorite__icon"
+                    />
+                  </div>
+                )}
+                {timeFooter && (
+                  <Element
+                    className="content job-posting-card__timeFooter"
+                    key="company"
+                    renderAs="p"
+                  >
+                    {timeFooter}
+                  </Element>
+                )}
+              </Element>
+            </Columns.Column>
+          </Columns>
```

Comment:
@rebecakipper Great stuff here, but we can also avoid using "Element" components from Bulma, in favor of using the html components directly. Eventually, we want to get rid of Bulma anyway so instead of using Element as p, or Element as img, please use p or img directly.

Reply:
Copy that! Will revert back to html